x-pack/plugins/security_solution/server/fleet_integration/notify_protection_feature_usage.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { NewPackagePolicy } from '../../../fleet/common';
+import { PolicyConfig, ProtectionModes } from '../../common/endpoint/types';
+import { EndpointMetadataService } from '../endpoint/services/metadata';
+import { FeatureUsageService } from '../endpoint/services/feature_usage/service';
+
+const OS_KEYS = ['windows', 'mac', 'linux'] as const;
```

Comment:
I'm surprise we don't already have a const for this. If not, I know for sure we have a Enum for it, so can you build it from that enum instead by taking its values?